src/app/webdav/blob-service.service.ts

```diff
@@ -0,0 +1,84 @@
+import { Injectable } from '@angular/core';
+import { AppConfig } from '../app-config/app-config';
+import webdav from 'webdav';
+import { DomSanitizer, SafeUrl } from '@angular/platform-browser';
+
+@Injectable({
+  providedIn: 'root'
```

Comment:
the `providedIn: 'root'` annotation overrides the `providers: []` statement in a module where you provide the service and always provides it in the root module. In that case we wouldn't need the BlobModule at all and can just use `providedIn: 'root'`. Otherwise we can remove the `providedIn` here and use the module to explicitly import it somewhere